source/features/faster-reviews.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import onReplacedElement from '../helpers/on-replaced-element';
+
+async function addSidebarReviewButton(): Promise<void> {
+	const reviewFormUrl = new URL(location.href);
+	reviewFormUrl.pathname += '/files';
+	reviewFormUrl.hash = 'submit-review';
+
+	select('[aria-label="Select reviewers"] .discussion-sidebar-heading')!.append(
+		<span style={{fontWeight: 'normal'}} className="js-reviews-container">
+			– <a href={reviewFormUrl.href} className="btn-link muted-link" data-hotkey="v">review</a>
+		</span>
+	);
+}
+
+async function initSidebar(): Promise<void> {
+	if (await elementReady('[aria-label="Select reviewers"] .discussion-sidebar-heading')) {
```

Comment:
Isn't this always available? Anyone can leave reviews.

You can probably merge `addSidebarReviewButton` with `initSidebar`